### PR TITLE
Add announcements link to installation script.

### PR DESCRIPTION
**Pull request by duftler**

### CODE REVIEW

File: scripts/install/setup.sh [hunk 1]
```diff
@@ -323,3 +323,6 @@ deploy_ready spin-deck "UI server"
 echo
 bold "Installation complete."
 echo
+bold "Make sure to sign up for Spinnaker for GCP updates and announcements:"
```
Comment (dorbin) [inline scripts/install/setup.sh]: Minor: s/Make sure to s/S/

Comment (duftler) [inline scripts/install/setup.sh]:
```suggestion
bold "Make sure to Sign up for Spinnaker for GCP updates and announcements:"
```

File: scripts/install/setup.sh [hunk 2]
```diff
@@ -323,3 +323,6 @@ deploy_ready spin-deck "UI server"
 echo
 bold "Installation complete."
 echo
+bold "Make sure to Sign up for Spinnaker for GCP updates and announcements:"
```
Comment (demobox) [inline scripts/install/setup.sh]: [minor] Perhaps just "Sign up for ..."? But fine either way.

Thanks!

Comment (duftler) [reply scripts/install/setup.sh]: Thought I had fixed that in response to David suggesting the same thing.

Got it right this time: https://github.com/GoogleCloudPlatform/spinnaker-for-gcp/pull/105

**Pull request merged**